azerothcore/azerothcore-wotlk
# fix(DB/creature): Ulduar 25 Antechamber trash pack

## CHANGES PROPOSED:
-  Update Antechamber trash packs in 10/25 Ulduar to mimic this retail video: https://youtu.be/m4fDOMYyjWw?t=654
-  Add lightning staff to iron menders

## ISSUES ADDRESSED:
- NPCs were crooked, wandering around, not at the ready position
- Iron Menders did not have weapons equipped


## TESTS PERFORMED:
- Applied SQL, checked positions and compared to retail video linked above
- [Before](https://user-images.githubusercontent.com/61223313/99886576-358eb100-2c03-11eb-9f37-8356d1aa49d2.png)
- [After](https://i.imgur.com/FxGMhJ3.jpg)
- [Iron Mender Staff](https://i.imgur.com/QRZhPQb.jpg)

## HOW TO TEST THE CHANGES:
- Apply SQL, create character
- Enter Ulduar, kill the first two bosses
- `.gm on` and check the NPCs, compare to the retail video linked above


## KNOWN ISSUES AND TODO LIST:
- The positioning is not exact.  I merely attempted to mimic the retail video linked above - if anyone has any actual data to pull from, please feel free to modify the sql.

## Target branch(es):
